app-libs/sgx-runtime/src/lib.rs

```diff
@@ -40,13 +40,16 @@ pub use evm::{
 
 use core::convert::{TryFrom, TryInto};
 use frame_support::weights::ConstantMultiplier;
+pub use itp_sgx_primitives::{
+	constants::SLOT_DURATION,
+	types::{AccountId, Address, Balance, BlockNumber, Hash, Header, Index, Signature},
+};
```

Comment:
Nice, I like the re-exports, but I would prefer that the re-exports are grouped separately.

Reply:
I don't really mind, but is there a reason why we import the types from `itp-sgx-primitives` instead of the `itp-sgx-runtime-primitives`?

Reply:
Thanks, nice catch, I renamed it in 4f295c7, to match the naming convention better.

Also moved the re-exports in 1cafbf6.